src/components/panel/panel.js

```diff
@@ -988,6 +1007,48 @@ MdPanelRef.prototype.focusOnOpen = function() {
 
 
 /**
+ * Shows the backdrop.
+ */
+MdPanelRef.prototype._showBackdrop = function() {
+  if (this._config.hasBackdrop) {
+    this._backdrop = this._$mdUtil.createBackdrop(this._$rootScope,
```

Comment:
Why isn't the backdrop just another pane instance? If that were the case, we could remove `$mdUtil.createBackdrop`, since that logic doesn't really have any business being there.